libraries/SDU/extras/SDUBoot/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh -x
+
+ARDUINO=arduino-cli
+SKETCH_NAME="SDUBoot.ino"
+SKETCH="$PWD/$SKETCH_NAME"
+BUILD_PATH="$PWD/build"
+OUTPUT_PATH="../../src/boot"
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+	ARDUINO="/Applications/Arduino.app/Contents/MacOS/Arduino"
```

Comment:
the original script from samd core expected on Linux that `arduino` is on PATH. I don't know why it had full path for Mac and if for ardoino-cli full path is required too. maybe it would be better to remove the `if`

Reply:
Ah, I see now:
https://github.com/arduino/ArduinoCore-samd/blob/master/libraries/SDU/extras/SDUBoot/build.sh

That script is configured to be used with IDE 1.x on all platforms.

>maybe it would be better to remove the if

I agree. I think it is OK for the script to assume the user has `arduino-cli` in their system PATH regardless of which OS they are using. If you want to make it more user friendly, you can do something like this:

```sh
if ! which arduino-cli &>/dev/null; then
  echo "Arduino CLI not found or not in PATH."
  echo "Please install: https://arduino.github.io/arduino-cli/latest/installation/"
  exit 1
fi
```

Reply:
this is definitely not something for an Arduino user. it is a tool for developers of Arduino. I didn't have arduino-cli in PATH and I had no problem to immediately identify while the script failed. so I just remove the `if`